code/modules/power/powernets/regional_powernet.dm

```diff
@@ -91,6 +105,19 @@
 
 /datum/regional_powernet/proc/calculate_queued_surplus()
 	return clamp(queued_power_production - queued_power_demand, 0, queued_power_production)
+
+/// Sets a new voltage type for the powernet, handles special behaviour when setting a new voltage
+/datum/regional_powernet/proc/update_voltage(new_voltage)
+	if(!new_voltage)
+		stack_trace("update_voltage called with a null new_voltage type")
+	if(!isnull(power_voltage_type) && new_voltage != power_voltage_type)
+		bzzzt(new_voltage)
+		return FALSE
+	power_voltage_type = new_voltage
+	return TRUE
+
+/datum/regional_powernet/proc/bzzzt(new_voltage)
```

Comment:
What does this do?

Reply:
was gonna be something with short circuits, decided to do something different and accidentally left this in. Remove in next commit.